test/integration/test_cloud_storage.py

```diff
@@ -0,0 +1,21 @@
+from exasol.secret_store import Secrets
+from exasol.github import retrieve_jar, Project
+from exasol.connections import open_bucketfs_connection, open_pyexasol_connection
+from exasol.bfs_utils import put_file
+from exasol.cloud_storage import setup_scripts
+
+from test.utils.integration_test_utils import setup_itde, get_script_counts
+
+
+def test_cloud_storage_setup_scripts(
+    secrets: Secrets,
+    setup_itde
+):
+    local_jar_path = retrieve_jar(Project.CLOUD_STORAGE_EXTENSION)
+    bucket = open_bucketfs_connection(secrets)
+    bfs_jar_path = put_file(bucket, local_jar_path)
+    assert str(bfs_jar_path).startswith("/buckets/")
+    with open_pyexasol_connection(secrets) as db_conn:
+        setup_scripts(db_conn, secrets.db_schema, str(bfs_jar_path))
```

Comment:
Do we have a test, that checks the scripts are generated accordingly to the bfs_jar_path, because in theory here the scripts could get generated, but are wrong

Reply:
Well, the scripts are trivial - just wrappers around jar methods. 

I can add unit tests for scripts generation, but I think that's an overkill.

Reply:
please create a ticket for the unit tests, we should do them, but it is probably not urgent

Reply:
Better will do in the same task, switching context for small things is not efficient 😃

Reply:
Could you also update the change log - chages_0.2.6.md - please?